deploy/helm/sumologic/templates/logs/collector/otelcol/daemonset.yaml

```diff
@@ -0,0 +1,109 @@
+{{ toYaml .Values.otellogs.daemonset.podAnnotations | indent 8 }}
+{{- end }}
+      labels:
+        app.kubernetes.io/name: {{ template "sumologic.labels.app.logs.collector.pod" . }}
+        {{- include "sumologic.labels.common" . | nindent 8 }}
+{{- if .Values.sumologic.podLabels }}
+{{ toYaml .Values.sumologic.podLabels | indent 8 }}
+{{- end }}
+{{- if .Values.otellogs.daemonset.podLabels }}
+{{ toYaml .Values.otellogs.daemonset.podLabels | indent 8 }}
+{{- end }}
+    spec:
+      securityContext:
+        {{- toYaml .Values.otellogs.daemonset.securityContext | nindent 8 }}
+      {{- if .Values.otellogs.daemonset.priorityClassName }}
+      priorityClassName: {{ .Values.metadata.logs.daemonset.priorityClassName | quote }}
+      {{- end }}
+      containers:
+      - args:
+        - --config=/etc/otelcol/config.yaml
+        image: {{ .Values.otellogs.image.repository }}:{{ .Values.otellogs.image.tag }}
+        imagePullPolicy: {{ .Values.otellogs.image.pullPolicy }}
+        name: otelcol
+        livenessProbe:
+          httpGet:
+            path: /
+            port: 13133 # Health Check extension default port.
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 13133 # Health Check extension default port.
+        volumeMounts:
+        - mountPath: /etc/otelcol
+          name: otelcol-config
+        - mountPath: /var/log/pods
+          name: varlogpods
+          readOnly: true
+        - mountPath: /var/lib/docker/containers
+          name: varlibdockercontainers
+          readOnly: true
+        - mountPath: {{ .Values.otellogs.config.extensions.file_storage.directory }}
+          name: file-storage
+        env:
+        - name: LOGS_METADATA_SVC
+          valueFrom:
+            configMapKeyRef:
+              name: sumologic-configmap
+              key: fluentdLogs
+        - name: NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        securityContext: 
+          {{- toYaml .Values.otellogs.daemonset.containers.otelcol.securityContext | nindent 10 }}
+      initContainers: # ensure the host path is owned by the otel user group
+      - name: changeowner
+        image: busybox
+        command:
+        - "sh"
+        - "-c"
+        - |
+          chown -R \
+            {{ .Values.otellogs.daemonset.securityContext.fsGroup }}:{{ .Values.otellogs.daemonset.securityContext.fsGroup }} \
+            {{ .Values.otellogs.config.extensions.file_storage.directory }}
+        volumeMounts:
+        - mountPath: {{ .Values.otellogs.config.extensions.file_storage.directory }}
+          name: file-storage
+      volumes:
+      - configMap:
+          defaultMode: 420
+          items:
+          - key: config.yaml
+            path: config.yaml
+          name: {{ template "sumologic.logs.collector.name.configmap" . }}
+        name: otelcol-config
+      - hostPath:
+          path: /var/log/pods
```

Comment:
Did we establish that we want to use `/var/log/pods` instead of [`/var/log/containers`](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/cddb1fc41d8fa277a069f1061657f2229c8dbbbe/deploy/helm/sumologic/values.yaml#L973)?

Reply:
It should be the same as for `Fluent Bit` IMO 🤔

Reply:
I'm not sure. @astencel-sumo prepared the OTC configuration originally, and it made sense to me as `/var/log/containers` is getting deprecated eventually afaik. Do you see any problems with using `/var/log/pods`?

Reply:
I don't have a strong preference but I would prefer to keep it the same as it is and **then** change it if need be later on.

Where did you see the info about it being deprecated?

Reply:
Basically, the official log path for CRI is `/var/log/pods`, and these are linked to `/var/log/containers` for backwards compatibility. I haven't been able to find an official deprecation timeline for the latter, was referring to [this comment](https://github.com/kubernetes/kubernetes/issues/53022#issuecomment-332150416).

I can switch it back to `/var/log/containers` for now, but it's going to be a bit annoying, as it exposes less metadata via the filepath, and I'll need to fiddle with the receiver operators again. 😒

Reply:
Ok, it's fine with me but I'd like this information to be easily discoverable by users, i.e. perhaps mention this in `values.yaml` somewhere next to the `otellogs` key?

Reply:
Allright, I'll add a comment then.